src/Quartz/Impl/StdSchedulerFactory.cs

```diff
@@ -508,11 +516,14 @@ protected virtual void ValidateConfiguration()
 
             Type tpType = loadHelper.LoadType(cfg.GetStringProperty(PropertyThreadPoolType)) ?? typeof(SimpleThreadPool);
 
+#if WINDOWS_THREADPOOL  // TODO (NetCore Port) - We can't use the ClrThreadPool on .NET Core because cross-plat threadpool implementations don't expose all the APIs we need
+                        // Use the SimpleThreadPool there for now and in the future create a new async-friendly threadpool that will work cross-plat.
             if (tpType == typeof(SimpleThreadPool))
```

Comment:
Yes, idea is to ditch the ClrThreadPool as soon as possible, but requires rewriting as you said. I probably need to learn more about the ".NET Core because cross-plat threadpool implementations don't expose all the APIs we need".